modules/tensor_mechanics/test/tests/j_integral_vtest/tests

```diff
@@ -85,4 +84,15 @@
                   'differentiation.'
     rel_err = 1.0e-4
   []
+  [j_int_bimaterial_sif]
+    type = 'CSVDiff'
+    input = 'j_int_bimaterial_sif.i'
+    csvdiff = 'j_int_bimaterial_sif_out.csv'
+    max_parallel = 1
+    requirement = 'The Domain Integral Action shall compute stress intensity factors from the '
+                  'interaction integral, with an added term stemming for spatially varying '
+                  'properties, that accounts for the crack perpendicularity to a bimaterial '
+                  'interface.'
```

Comment:
Suggested rewording:
The Domain Integral Action shall compute stress intensity factors from the interaction integral for a crack perpendicular to a bi-material interface, treating the interface as a functionally-graded material

Reply:
Used that wording and added a test that I used to verify against literature numerical data.